documentation-site/pages/blog/responsive-web/index.mdx

```diff
@@ -0,0 +1,248 @@
+
+```js
+const LoadStatus = themedStyled('span', ({$theme}) => ({
+  ...$theme.typography.font350,
+  color: $theme.colors.mono800,
+  [$theme.media.small]: {
+    ...$theme.typography.font500,
+    color: $theme.colors.mono900,
+  },
+}));
+```
+
+With our small breakpoint of 769px, this means that the component will change font and color as the screen increases past 769px:
+
+<BlogImage
+  src={responsiveLoadStatusGif}
+  alt="resizing the window smaller and seeing the pink-highlighted text change size and color"
+  caption="LoadStatus component, highlighted in pink, changes font and color on mobile"
+/>
+
+## Block responsive API
+
+The [Block](/components/block/#responsive-layouts) component accepts props either as a single value or an array of values. When an array is used, it will generate media queries based on the current theme breakpoints.
+
+This gives us a simpler way of creating components:
+
+```js
+const NavBar = props => (
+  <Block
+    height={['scale1400', 'scale1600']}
+    paddingLeft={['scale800', 'scale1200']}
+    paddingRight={['scale800', 'scale1200']}
+    {...props}
+  />
+);
+
+// is equivalent to
+
+const NavBar = themedStyled('div', ({$theme}) => ({
+  height: $theme.sizing.scale1400,
+  paddingRight: $theme.sizing.scale800,
+  paddingLeft: $theme.sizing.scale800,
+  [$theme.media.small]: {
+    height: $theme.sizing.scale1600,
+    paddingRight: $theme.sizing.scale1200,
+    paddingLeft: $theme.sizing.scale1200,
+  },
+}));
+```
+
+<BlogImage
+  src={responsiveNavBarGif}
+  alt="resizing the window smaller and seeing the yellow-highlighted nav bar change height and padding"
+  caption="NavBar component, highlighted in yellow, changes height and padding on mobile"
+/>
+
+## Mobile-only or desktop-only Blocks
+
+We can also take advantage of these responsive properties to completely hide a Block component using CSS `display: none`.
+
+```js
+// NavLink appears only on desktop
+const NavLink = props => <Block display={['none', 'block']} {...props} />;
+
+// TabLink appears only on desktop
+const TabLink = props => <Block display={['block', 'none']} {...props} />;
+```
+
+<BlogImage
+  src={responsiveHideGif}
+  alt="resizing the window smaller and seeing the green-highlighted components appear or disappear"
+  caption="Mobile/desktop-only components, highlighted in green, appear or disappear on mobile"
+/>
+
+## FlexGrid component
+
+Another handy tool for building responsive UIs is the [FlexGrid](/components/flex-grid/) component ([v6.19.0](https://github.com/uber-web/baseui/releases/tag/v6.19.0)+). This component lets you build grid layouts using CSS flexbox, supporting Internet Explorer 10+ users. It sets gaps between the columns and rows using `:nth-child` and `:nth-last-child` math, and also allows for responsive prop arrays like the [Block](/components/block/#responsive-layouts) component.
```

Comment:
Add a link to the file with css math magic here. It's really cool so you should show it off more!

Reply:
Added link and added more explanation